couchdb-cleanup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/zsh
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
Not everyone has zsh installed

Reply:
Oops, I didn't even mean to commit this and couchdb-init.sh. These were just helper scripts for my environment.

Is there a way to remove them from the pull request?

Reply:
Actually I like them :)

Maybe you can fix that and move them in a dedicated folder?

Reply:
They are quite handy but shouldn't be part of this pull request. I'll go ahead and create another one for that ;)

Reply:
Please do it! :)